src/table/classes.scss

```diff
@@ -28,6 +28,14 @@
 
 .iui-paginator {
   @include iui-paginator;
+
+  .iui-paginator-ellipsis {
```

Comment:
> Ellipsis class could go on the root level as there are no other rules

So move it to root?

> paginator is pretty specific and I dont think we expect lots of overrides there

Keeping the specificity low always helps. We could in the future allow using this paginator outside the table

Reply:
moved to the root